src/devices/cpu/h8/gt913.cpp

```diff
@@ -88,7 +88,14 @@ void gt913_device::device_add_mconfig(machine_config &config)
 {
 	GT913_INTC(config, "intc");
 
-	GT913_SOUND_HLE(config, m_sound, 0);
+	/*
+	generate sound at 104 cycles per sample (= 144.231 kHz sample clock to the DAC)
+	on keyboard models that include a DSP, this also results in a multiple
+	of the 36.058 kHz CPU->DSP sync signal shown in some schematics (WK-1200 and others)
+	*/
+	GT913_SOUND(config, m_sound, std::round(clock() / 104.0f));
```

Comment:
I'm too late, since PR was merged(and reverted).
But FYI this sample clock divider is normally done in stream_alloc and, like: m_stream = stream_alloc(0, 2, clock() / 104);

Reply:
Also, you’re supposed to use `DERIVED_CLOCK` for clocks that are derived from the parent device clock.